File: core/src/main/java/com/pluxity/facility/building/dto/BuildingCreateRequest.java
package com.pluxity.facility.building.dto;

import com.pluxity.facility.dto.FacilityCreateRequest;
import com.pluxity.facility.floor.dto.FloorRequest;
import java.util.List;

public record BuildingCreateRequest(FacilityCreateRequest facility, List<FloorRequest> floors) {}

File: core/src/main/java/com/pluxity/building/dto/BuildingCreateRequest.kt
package com.pluxity.building.dto

import com.pluxity.facility.dto.FacilityCreateRequest
import com.pluxity.facility.floor.dto.FloorRequest

data class BuildingCreateRequest(
    val facility: FacilityCreateRequest,
    val floors: List<FloorRequest>,
)
